apps/backend/src/lambdas/reconcile.ts

```diff
@@ -130,14 +129,10 @@ const runCashReconciliation = async (
       appLogger.log({ dateRange }, CashReconciliationService.name);
 
       appLogger.log(
-        `Processing Cash Reconciliation for: ${location.description} (${location.location_id}) for ${dateRange.from_date} - ${dateRange.to_date}`,
-        CashReconciliationService.name
+        `Processing Cash Reconciliation for: ${location.description} (${location.location_id}) for ${dateRange.from_date} - ${dateRange.to_date}`
```

Comment:
I will try to avoid the erratic use of caps lock in the future. Maybe we can have a standard format for how to write the logs?